.github/workflows/cli-release-process.yml

```diff
@@ -7,24 +7,137 @@ on:
         required: true
       next:
         description: 'Next version'
+        default: '1.0.0-SNAPSHOT'
```

Comment:
If we are going to follow the process where we maintain the 1.0.0-SNAPSHOT version in the pom, even though this is default may be better to not have this as an option.